test/view_component/view_component_test.rb

```diff
@@ -26,11 +26,49 @@ def test_renders_content_from_block
   def test_render_without_template
     render_inline(InlineComponent.new)
 
-    assert_predicate InlineComponent, :inlined?
+    assert_predicate InlineComponent, :all_inlined?
     assert_not_predicate InlineComponent, :compiled?
```

Comment:
To me, `compiled?` simply means that the component has been fully initialized / optimized to render, so perhaps it would make more sense for it to be true in this case?